From c8d32248b8fbdc350eaeee23051c30ca46662dc7 Mon Sep 17 00:00:00 2001
From: Axel Lin <axel.lin@ingics.com>
Date: Wed, 26 Jun 2013 00:25:02 +0800
Subject: pwm: renesas-tpu: Add MODULE_ALIAS to make module auto loading work

This driver can be built as module, add MODULE_ALIAS to make module auto loading
work.

Signed-off-by: Axel Lin <axel.lin@ingics.com>
Signed-off-by: Thierry Reding <thierry.reding@gmail.com>
(cherry picked from commit 71077bc8db67f560453d62c48e3bec467ec6b37f)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 drivers/pwm/pwm-renesas-tpu.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pwm/pwm-renesas-tpu.c b/drivers/pwm/pwm-renesas-tpu.c
index 03c1aa3c..26008927 100644
--- a/drivers/pwm/pwm-renesas-tpu.c
+++ b/drivers/pwm/pwm-renesas-tpu.c
@@ -471,3 +471,4 @@ module_platform_driver(tpu_driver);
 MODULE_AUTHOR("Laurent Pinchart <laurent.pinchart@ideasonboard.com>");
 MODULE_DESCRIPTION("Renesas TPU PWM Driver");
 MODULE_LICENSE("GPL v2");
+MODULE_ALIAS("platform:renesas-tpu-pwm");
-- 
1.8.4.3.gca3854a

